src/androidTest/java/com/mixpanel/android/viewcrawler/ViewSnapshotTest.java

```diff
@@ -32,11 +31,11 @@ public void setUp() throws NoSuchMethodException {
 
         final List<PropertyDescription> props = new ArrayList<PropertyDescription>();
 
-        final Caller textGetter = new Caller(TextView.class, "getText", new Object[0], CharSequence.class);
+        final PropertyCaller textGetter = new PropertyCaller(TextView.class, "getText", new Object[0], CharSequence.class);
```

Comment:
Caller isn't really a type of change, and it isn't limited to properties- you can use callers to call any method. It's what the property visitor uses, but it isn't a visitor itself.